site/content/api/Event-API.md

```diff
@@ -31,7 +31,7 @@ Hono supports *AT LEAST ONCE* delivery of *Event* messages only. A client theref
 
 **Message Flow**
 
-TBD
+![Upload event data flow](../uploadEvent_Success.png)
```

Comment:
can you also add the textual description (analogously to Telemetry API)?

when doing so, please explicitly point out that, in contrast to the Telemetry case, the protocol adapter waits for the `accepted` disposition to arrive before signaling the device that it has accepted the event.